internal/configuration/env_config.go

```diff
@@ -164,63 +166,63 @@ func (c *EnvConfig) GetPostgresConnectionMaxOpen() int {
 	return i
 }
 
-// GetIdlerURL returns the Idler API URL as set via default, config file, or environment variable
+// GetIdlerURL returns the Idler API URL as set via default, config file, or environment variable.
 func (c *EnvConfig) GetIdlerURL() string {
 	callPtr, _, _, _ := runtime.Caller(0)
 	value := getConfigValueFromEnv(util.NameOfFunction(callPtr))
 
 	return value
 }
 
-// GetAuthURL returns the Auth API URL as set via default, config file, or environment variable
+// GetAuthURL returns the Auth API URL as set via default, config file, or environment variable.
 func (c *EnvConfig) GetAuthURL() string {
 	callPtr, _, _, _ := runtime.Caller(0)
 	value := getConfigValueFromEnv(util.NameOfFunction(callPtr))
 
 	return value
 }
 
-// GetTenantURL returns the F8 Tenant API URL as set via default, config file, or environment variable
+// GetTenantURL returns the F8 Tenant API URL as set via default, config file, or environment variable.
 func (c *EnvConfig) GetTenantURL() string {
 	callPtr, _, _, _ := runtime.Caller(0)
 	value := getConfigValueFromEnv(util.NameOfFunction(callPtr))
 
 	return value
 }
 
-// GetWitURL returns the WIT API URL as set via default, config file, or environment variable
+// GetWitURL returns the WIT API URL as set via default, config file, or environment variable.
 func (c *EnvConfig) GetWitURL() string {
 	callPtr, _, _, _ := runtime.Caller(0)
 	value := getConfigValueFromEnv(util.NameOfFunction(callPtr))
 
 	return value
 }
 
-// GetKeycloakURL returns the Keycloak API URL as set via default, config file, or environment variable
+// GetKeycloakURL returns the Keycloak API URL as set via default, config file, or environment variable.
 func (c *EnvConfig) GetKeycloakURL() string {
 	callPtr, _, _, _ := runtime.Caller(0)
 	value := getConfigValueFromEnv(util.NameOfFunction(callPtr))
 
 	return value
 }
 
-// GetAuthToken returns the Auth token as set via default, config file, or environment variable
+// GetAuthToken returns the Auth token as set via default, config file, or environment variable.
 func (c *EnvConfig) GetAuthToken() string {
 	callPtr, _, _, _ := runtime.Caller(0)
 	value := getConfigValueFromEnv(util.NameOfFunction(callPtr))
 
 	return value
 }
 
-// GetRedirectURL returns the redirect url to be passed to Auth as set via default, config file, or environment variable
+// GetRedirectURL returns the redirect url to be passed to Auth as set via default, config file, or environment variable.
```

Comment:
:-) Nice to see that you take care of punctuation as well

Reply:
This is mostly to keep you happy ;) 
Otherwise, all of my comments end without full stop